Fang-4-Group/Graduation-Project
# Merge yunsi/GP-36/store-img-to-db branch into master

## Changes
- Add three api for testing img storing
- Add api testing in test_route.py
- add *.pyc to .gitignore

## Testing Steps
1. pull this branch
2. run `docker-compose up -d`
3. click http://localhost:7877/test_create_img/
    If you see the message below, try the next step.
    ```
    {
        "status": 200,
        "message": "success to create table"
    }
    ```
 4. click http://localhost:7877/show_img/
     If you see the image below showing on your browser, then the way to store image is ok.
    <img width="448" alt="image" src="https://github.com/Fang-4-Group/Graduation-Project/assets/93365070/6577bd64-8ef5-44e0-8fd8-bb8709044166">
5. click http://localhost:7877/test_drop_img/ to drop the table storing img info in posgreSQL
    If you see the message below, the testing end.
    ```
    {
        "status": 200,
        "message": "success to drop table"
    }
    ```
